--- mysql_install_db    2018-03-15 03:21:07.862269654 +0000
+++ /usr/bin/mysql_install_db   2018-01-18 13:32:55.000000000 +0000
@@ -583,7 +583,8 @@
                                     "libexec","sbin","bin") ||  # ,"sql"
                     find_in_basedir($opt,"file","mysqld-nt",
                                   "bin");  # ,"sql"
-  $pkgdatadir     = '/usr/share/mysql-community-server';
+  $pkgdatadir     = find_in_basedir($opt,"dir","fill_help_tables.sql",
+                                    "share","share/mysql");  # ,"scripts"
   $scriptdir      = "$opt->{basedir}/scripts";
 }
 else
@@ -746,6 +747,7 @@
 # ----------------------------------------------------------------------
 # Create the system and help tables by passing them to "mysqld --bootstrap"
 # ----------------------------------------------------------------------
+
 report_verbose_wait($opt,"Installing MySQL system tables...");
 
 open(SQL, $create_system_tables)
